dotnet-datetime/TimeSpanInCSharp/TimeSpanInCSharpTests/TimeSpanUnitTests.cs

```diff
@@ -0,0 +1,201 @@
+        {
+            var firstTimeSpan = new TimeSpan(2, 60, 3600);
+            var secondTimeSpan = new TimeSpan(1, 30, 1800);
+
+            var divisionByMethod = _timeSpanMethods.DivideTimeSpanWithDivisor(firstTimeSpan, 2);
+            var divisionByOperator = firstTimeSpan / 2;
+
+            Assert.AreEqual(divisionByMethod, secondTimeSpan);
+            Assert.AreEqual(divisionByOperator, secondTimeSpan);
+            Assert.IsInstanceOfType(divisionByMethod, typeof(TimeSpan));
+            Assert.IsInstanceOfType(divisionByOperator, typeof(TimeSpan));
+        }
+
+        [TestMethod]
+        public void GivenTwoTimeSpanValues_WhenEqualityOperationApplied_VerifyAccurateResult()
+        {
+            var firstTimeSpan = new TimeSpan(1, 60, 3600);
+            var secondTimeSpan = new TimeSpan(1, 60, 3600);
+            var thirdTimeSpan = new TimeSpan(3, 60, 4000);
+
+            Assert.IsTrue(firstTimeSpan == secondTimeSpan);
+            Assert.IsTrue(firstTimeSpan.Equals(secondTimeSpan));
+            Assert.IsTrue(firstTimeSpan < thirdTimeSpan);
+            Assert.IsTrue(thirdTimeSpan > secondTimeSpan);
+            Assert.IsTrue(thirdTimeSpan >= secondTimeSpan);
+            Assert.IsTrue(firstTimeSpan <= thirdTimeSpan);
+            Assert.IsTrue(firstTimeSpan != thirdTimeSpan);
+        }
+
+        [TestMethod]
+        public void GivenATimeSpanAndFactor_WhenMultiplied_VerifyAccurateResult()
+        {
+            var expected = new TimeSpan(2, 60, 3600);
+            var firstTimeSpan = new TimeSpan(1, 30, 1800);
+            var factor = 2;
+
+            var multiplyMethod = _timeSpanMethods.MultiplyTimeSpan(firstTimeSpan, factor);
+            var multiplyOperator = firstTimeSpan * factor;
+
+            Assert.AreEqual(multiplyMethod, multiplyOperator);
+            Assert.AreEqual(expected, multiplyOperator);
+            Assert.AreEqual(expected, multiplyMethod);
+            Assert.IsInstanceOfType(multiplyMethod, typeof(TimeSpan));
+            Assert.IsInstanceOfType(multiplyOperator, typeof(TimeSpan));
+        }
+
+        [TestMethod]
+        public void GivenTwoTimeSpans_WhenSubtractOperationApplied_VerifyAccurateResult()
+        {
+            var firstTimeSpan = new TimeSpan(2, 60, 3600);
+            var secondTimeSpan = new TimeSpan(1, 30, 1800);
+            var expected = new TimeSpan(1, 30, 1800);
+
+            var subtractMethod = _timeSpanMethods.SubtractTimeSpan(firstTimeSpan, secondTimeSpan);
+            var subtractOperator = firstTimeSpan - secondTimeSpan;
+
+            Assert.AreEqual(subtractMethod, subtractOperator);
+            Assert.AreEqual(expected, subtractOperator);
+            Assert.AreEqual(expected, subtractMethod);
+            Assert.IsInstanceOfType(subtractMethod, typeof(TimeSpan));
+            Assert.IsInstanceOfType(subtractOperator, typeof(TimeSpan));
+        }
+
+        [TestMethod]
+        public void GivenATimeSpan_WhenConvertedToString_VerifyAccurateResult()
```

Comment:
We should verify the format of the returned value, so something like:

```
var expected = "04:00:00";

Assert.AreEqual(expected, actual);
```

This will need added to the article also.